mlflow/store/artifact/databricks_logged_model_artifact_repo.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import posixpath
+import re
+from concurrent.futures import Future, ThreadPoolExecutor
+from pathlib import Path
+from typing import Optional
+
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.config import Config
+from databricks.sdk.errors.platform import NotFound
+from databricks.sdk.service.files import FilesAPI
+
+from mlflow.entities import FileInfo
+from mlflow.exceptions import MlflowException
+from mlflow.store.artifact.artifact_repo import ArtifactRepository
+from mlflow.store.artifact.databricks_artifact_repo import DatabricksArtifactRepository
+
+_logger = logging.getLogger(__name__)
+
+
+class DatabricksLoggedModelArtifactRepository(ArtifactRepository):
+    """
+    Artifact repository for interacting with logged model artifacts in a Databricks workspace.
+    If operations using the Databricks SDK fail for any reason, this repository automatically
+    falls back to using the `DatabricksArtifactRepository`, ensuring operational resilience.
+    """
+
+    # Matches URIs of the form:
+    # databricks/mlflow-tracking/<experiment_id>/logged_models/<model_id>/<relative_path>
+    _URI_REGEX = re.compile(
+        r"databricks/mlflow-tracking/(?P<experiment_id>[^/]+)/logged_models/(?P<model_id>[^/]+)(?P<relative_path>/.*)?$"
+    )
+
+    @staticmethod
+    def is_logged_model_uri(artifact_uri: str) -> bool:
+        return bool(DatabricksLoggedModelArtifactRepository._URI_REGEX.search(artifact_uri))
+
+    def __init__(self, artifact_uri: str) -> None:
+        super().__init__(artifact_uri)
+        m = self._URI_REGEX.search(artifact_uri)
+        if not m:
+            raise MlflowException.invalid_parameter_value(
+                f"Invalid artifact URI: {artifact_uri}. Expected URI of the form "
+                f"databricks/mlflow-tracking/<EXP_ID>/logged_models/<MODEL_ID>"
+            )
+        experiment_id = m.group("experiment_id")
+        model_id = m.group("model_id")
+        relative_path = m.group("relative_path") or ""
+        self.root_path = f"/Mlflow/Artifacts/{experiment_id}/LoggedModels/{model_id}{relative_path}"
```

Comment:
```suggestion
        self.root_path = f"WorkspaceInternal/Mlflow/Artifacts/{experiment_id}/LoggedModels/{model_id}{relative_path}"
```

we changed this prefix name